README.md

```diff
@@ -3,24 +3,35 @@ It relies on Spack and the builtin Spack packages, some of which may be overridd
 
 ## Getting started
 
-Initial setup for bash:
+Initial setup for bash-like shells:
 
+```
 cd /path/your-working-dir
-
 git clone https://github.com/spack/spack.git
-
 git clone https://github.com/HDFGroup/hdf-spack
-
 source ./spack/share/spack/setup-env.sh
-
 spack repo add ./hdf-spack
-
 spack repo list
+```
+
+This repo's packages are now available locallay and have precidence over Spack's builtin repo.

```

Comment:
This repo's packages are now available locally and have precedence over Spack's builtin repo.
```suggestion
This repo's packages are now available locallay and have precidence over Spack's builtin repo.

```

Reply:
Suggestion was supposed to change locallay to locally and precidence to precedence.  Not sure why it didn't.